easybuild/easyblocks/t/tensorflow.py

```diff
@@ -421,10 +421,12 @@ def setup_build_dirs(self):
     def configure_step(self):
         """Custom configuration procedure for TensorFlow."""
 
-        binutils_root = get_software_root('binutils')
-        if not binutils_root:
-            raise EasyBuildError("Failed to determine installation prefix for binutils")
-        self.binutils_bin_path = os.path.join(binutils_root, 'bin')
+        # determine location where binutils' ld command is installed
+        # note that this may be an RPATH wrapper script (when EasyBuild is configured with --rpath)
+        ld_path = which('ld')
+        self.log.info("Found ld at %s", ld_path)
+        self.binutils_bin_path = os.path.dirname(ld_path)
+        self.log.info("Assuming that all binutils tools are located at %s", self.binutils_bin_path)
```

Comment:
Couldn't we check instead of assume?

Reply:
Actually this log message is misleading, because when `--rpath` is used actually only `ld` (a wrapper script) will be located there, but that's enough...

I'll just remove the log message.